tests/unit/preprocessor/_time/test_time.py

```diff
@@ -1781,7 +1781,7 @@ def test_get_time_weights_lazy():
     weights = get_time_weights(cube)
     assert isinstance(weights, da.Array)
     assert weights.shape == (2, )
-    assert weights.chunks == ((1, 1), )
+    assert weights.chunks == ((2, ), )
```

Comment:
@valeriupredoi This looks wrong, the weights should have 2 chunks, see #2404

Reply:
eh well in that case there's something :fish: y in the weights calculation?